src/Umbraco.Web.UI.Client/src/common/directives/components/umbminisearch.component.js

```diff
@@ -11,25 +11,29 @@
                 model: "=",
                 onStartTyping: "&?",
                 onSearch: "&?",
-                onBlur: "&?"
+                onBlur: "&?",
+                labelKey: "@?"
             }
         });
 
-    function UmbMiniSearchController($scope) {
+    function UmbMiniSearchController($scope, localizationService) {
         
         var vm = this;
 
         vm.onKeyDown = onKeyDown;
         vm.onChange = onChange;
+        vm.inputId = randomUniquId();
```

Comment:
How about this here, where the component has a `inputId` property like in in checkbox and radiobutton components?

```
vm.inputId = vm.inputId || "umb-mini-search_" + String.CreateGuid();
```